### PR TITLE
More changes toward trusty.

### PR DESCRIPTION
**Pull request by obino**
By default flower and monit shouldn't be exposed, since they are mainly debugging tools.

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/553/

File: AppServer_Java/build.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
   </target>
 
   <target name="extract-sdk" unless="sdk.exists">
-    <get src="${gae_url}" dest="appengine-java-sdk.zip" verbose="true" />
+    <get src="${gae_url}" dest="appengine-java-sdk.zip" verbose="false" />
```
Comment (menivaitsi) [inline AppServer_Java/build.xml]: The original SDK is still needed, I believe.

Comment (obino) [inline AppServer_Java/build.xml]: This just make it not verbose. It's easier to look at out build logs.

Comment (menivaitsi) [inline AppServer_Java/build.xml]: Right, I missed that. Thanks.

File: firewall.conf [hunk 2]
```diff
@@ -15,9 +15,9 @@ iptables -A INPUT -p tcp --dport ssh -j ACCEPT
 # Allow for all traffic from anywhere for the following services
 iptables -A INPUT -p tcp --dport 80 -j ACCEPT          # nginx
 iptables -A INPUT -p tcp --dport 443 -j ACCEPT         # nginx (SSL)
-iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
+#iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
```
Comment (menivaitsi) [inline firewall.conf]: Disabling those services requires AppDashboard modifications as well.
We'll need to disable the respective sidebar tabs, for example.

Comment (obino) [inline firewall.conf]: Well, should we just leave them for when the admin will enable those services? Or just mention in the dashboard that they need to be enabled?

Comment (menivaitsi) [inline firewall.conf]: I'd, at least, disable the dashboard buttons (without hiding them).
Otherwise, if you click, you get ERR_CONNECTION_TIMED_OUT.

Comment (obino) [inline firewall.conf]: Perhaps we can add some notification around the button that tells that those features needs to be enabled by the cloud administrator?

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/618/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/620/

**Pull request merged**